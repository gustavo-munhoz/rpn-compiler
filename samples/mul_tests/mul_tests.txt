(3 3 *)
(2 40 *)
(10 0.5 *)
(0.5 0.5 *)
(-2 3 *)
(-4 -5 *)
(100 0 *)
(0 -20 *)
(0 0 *)
(12.5 1 *)
(12.5 -1 *)
(512 256 *)
(1000 100 *)
(-1000 100 *)
(-1000 -100 *)
(600 600 *)
(0.5 0.00006103515625 *)
(0.0001220703125 0.0001220703125 *)
(0.0078125 0.0078125 *)
(0.00048828125 0.00048828125 *)