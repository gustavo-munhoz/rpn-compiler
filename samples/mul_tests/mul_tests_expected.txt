0x4880
0x5500
0x4500
0x3400
0xC600
0x4D00
0x0000
0x0000
0x0000
0x4A40
0xCA40
0x7C00
0x7C00
0xFC00
0x7C00
0x7C00
0x0000
0x0000
0x0400
0x0000